app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -469,21 +473,7 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
 
     @OnTextChanged(value = R.id.document__fragment__edit__highlighting_editor, callback = OnTextChanged.Callback.TEXT_CHANGED)
     public void onContentEditValueChanged(CharSequence text) {
-        if ((_lastChangedThreadStart + HISTORY_DELTA) < System.currentTimeMillis()) {
-            _lastChangedThreadStart = System.currentTimeMillis();
-            _hlEditor.postDelayed(() -> {
-                _document.setContent(text.toString());
-                Activity activity = getActivity();
-                if (activity != null && activity instanceof AppCompatActivity) {
-                    ((AppCompatActivity) activity).supportInvalidateOptionsMenu();
-                }
-            }, HISTORY_DELTA);
-        }
-        Activity activity = getActivity();
-        if (activity != null && activity instanceof AppCompatActivity) {
-            ((AppCompatActivity) activity).supportInvalidateOptionsMenu();
-        }
-
```

Comment:
It seems that the options menu for the current activity is invalidated immediately and then again after 5 seconds ? @gsantner Could you help me understand what is being done here?